diff --git a/lib/credentials.js b/lib/credentials.js
--- a/lib/credentials.js
+++ b/lib/credentials.js
@@ -42,5 +42,5 @@ export const fromOAuthToken = (input) => Layer.succeed(Credentials, Effect.succee
 const envConfig = EffectConfig.all({
-    tokenId: EffectConfig.string("PLANETSCALE_API_TOKEN_ID"),
-    token: EffectConfig.string("PLANETSCALE_API_TOKEN"),
-    organization: EffectConfig.string("PLANETSCALE_ORGANIZATION"),
+    tokenId: EffectConfig.String("PLANETSCALE_API_TOKEN_ID"),
+    token: EffectConfig.String("PLANETSCALE_API_TOKEN"),
+    organization: EffectConfig.String("PLANETSCALE_ORGANIZATION"),
 });
diff --git a/src/credentials.ts b/src/credentials.ts
--- a/src/credentials.ts
+++ b/src/credentials.ts
@@ -91,5 +91,5 @@ export const fromOAuthToken = (input: OAuthTokenInput): Layer.Layer<Credentials>
 const envConfig = EffectConfig.all({
-  tokenId: EffectConfig.string("PLANETSCALE_API_TOKEN_ID"),
-  token: EffectConfig.string("PLANETSCALE_API_TOKEN"),
-  organization: EffectConfig.string("PLANETSCALE_ORGANIZATION"),
+  tokenId: EffectConfig.String("PLANETSCALE_API_TOKEN_ID"),
+  token: EffectConfig.String("PLANETSCALE_API_TOKEN"),
+  organization: EffectConfig.String("PLANETSCALE_ORGANIZATION"),
 });
